provider/app.js

```diff
@@ -83,15 +80,15 @@ function createTriggerDb () {
   // no need for a promise here, but leaving code inplace until we prove out the question of cookie usage
```

Comment:
lets fix this up if no promise is needed.  we are not going to use cookies for auth